### PR TITLE
[s] Fixed ORM exploit

### PR DESCRIPTION
**Pull request by Thunder12345**
:cl: Thunder12345
fix: You can no longer extract negative sheets from the ORM
/:cl:

### CODE REVIEW

File: code/modules/mining/machine_redemption.dm [hunk 1]
```diff
@@ -235,7 +235,7 @@
 			var/obj/item/stack/sheet/out = new inp.type(src, 0, FALSE)
 			var/desired = input("How many sheets?", "How many sheets to eject?", 1) as null|num
 			out.amount = round(min(desired,50,inp.amount))
```
Comment (lzimann) [inline code/modules/mining/machine_redemption.dm]: i was just going to throw an `abs()` here, but I guess this works too

**Comment by Tacolizard** [comment]
this is hilarious. how did the orm have so many exploits? I've never heard of anyone exploiting it

**Comment by lzimann** [comment]
This one is my fault when I refactored the orm code, and forgot about negatives.

**Comment by octareenroon91** [comment]
Refactors are so thankless exploits might not even get exploited.

**Comment by ExcessiveUseOfCobblestone** [comment]
can you add an `else to_chat(user, "<span class='notify'> Invalid Input, please try again.</span>")` or something that indicates it's not doing anything because of the input?

**Comment by Thunder12345** [comment]
I think it should be obvious releasing negative amounts shouldn't work

**Pull request merged**